lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -2190,8 +2185,8 @@ def self.notify
           :allowed_from_server => true,
           # Keep the extra two-space indent before the second bullet to appease translation tool
           :description => <<~DESC
-            * Defines the maximum number of span events reported from a single harvest. Any Integer between `1` and `10000` is valid.'
-              * When configuring the agent for [AI monitoring](/docs/ai-monitoring/intro-to-ai-monitoring), set to max value `10000`.\
+            - Defines the maximum number of span events reported from a single harvest. Any Integer between `1` and `10000` is valid.'
+            - When configuring the agent for [AI monitoring](/docs/ai-monitoring/intro-to-ai-monitoring), set to max value `10000`.\
```

Comment:
I think we lost the little space here:
```suggestion
              - When configuring the agent for [AI monitoring](/docs/ai-monitoring/intro-to-ai-monitoring), set to max value `10000`.\
```

Reply:
Oh thank you! Reverted that change https://github.com/newrelic/newrelic-ruby-agent/pull/3102/commits/58608a13b7eafb59cfb6b860aa3ab87a5ff23dfe